crates/holochain_sqlite/Cargo.toml

```diff
@@ -67,10 +63,10 @@ rand = "0.8.5"
 
 [build-dependencies]
 pretty_assertions = "1.4"
-sqlformat = "0.1.6"
+sqlformat = "0.2"
 
 [target.'cfg(loom)'.dev-dependencies]
-loom = { version = "0.6", features = ["futures", "checkpoint"] }
+loom = { version = "0.7", features = ["futures", "checkpoint"] }
```

Comment:
Loom tests don't currently work. I didn't think were were getting much value from them because they can really only test limited primitives and not functional code like `holochain_sqlite`. I'd probably prefer removing this